docs/pages/apis/fields.mdx

```diff
@@ -310,6 +311,7 @@ export default config({
         fieldName: password({
           isRequired: true,
           minLength: 10,
```

Comment:
Maybe we should drop this [back to 8](https://github.com/keystonejs/keystone-5/blob/main/packages/fields/src/types/Password/Implementation.js#L25) now that `rejectCommon` back and defaults to being enabled? Length is a lot less critical if we're blocking dumb stuff.

```suggestion
          minLength: 8,
```

Reply:
The default is 8, that is showing an example of configuring it differently